charts/corda-dev-prereqs/values.yaml

```diff
@@ -13,9 +13,16 @@ maxConnections: 200
 # Kafka specific configuration
 kafka:
   # -- Resource requests/limits for Kafka
-  resources: {}
+  resources:
+    requests:
```

Comment:
The formatting here is wrong as `limit` is a peer of `requests`. Also, in the developer context in which this chart exists, it is typically not helpful to specify requests/limits. At least on my machine, I only have a single node with a fixed amount of resource, and specifying these values is only likely to mean I can't deploy something that might otherwise fit.

Reply:
Youw ere of course right around the syntax, I fixed that. I'm really trying to figure out what values we use for Kafka on the e2e tests on Jenkins.

Reply:
"use" or "should use" (given I see you've found the ticket saying that no limits are set)? Also, remember that the E2E tests are not using the `corda-dev-prereqs` chart. They're using the version on the `corda-dev` branch.

Reply:
now targetting corda-dev